reference_simulations/catalog_files/MOON.txt

```diff
@@ -0,0 +1,150 @@
+SOURCE_ID	RA_ICRS [deg]	Dec_ICRS [deg]	Flux [Jy]	Frequency [Hz]
```

Comment:
Looks like there's a few files still here in the reference_simulations folder that are duplicated in the src/pyuvsim/data folder.